src/app/util/message.cpp

```diff
@@ -182,11 +182,11 @@ uint8_t * emberAfPutDateInResp(EmberAfDate * value)
 // buffer)
 // ------------------------------------
 
-// retrieves an uint32_t which contains between 1 and 4 bytes of relevent data
+// retrieves an uint64_t which contains between 1 and 8 bytes of relevent data
```

Comment:
```suggestion
// retrieves an uint64_t which contains between 1 and 8 bytes of relevant data
```
Preexisting, I know.